src/components/footer/Footer.tsx

```diff
@@ -38,17 +37,124 @@ export function Footer(props: FooterProps) {
 	} = props.footer || DEFAULT_FOOTER;
 
 	return (
-		<FooterDSFR
-			accessibility={accessibility}
-			brandTop={<ConvertContent content={brandTop as any} />}
-			homeLinkProps={homeLinkProps}
-			termsLinkProps={termsLinkProps}
-			websiteMapLinkProps={websiteMapLinkProps}
-			license={<ConvertContent content={license as any} />}
-			personalDataLinkProps={personalDataLinkProps}
-			contentDescription={contentDescription}
-			operatorLogo={operatorLogo}
-			bottomItems={bottomItems}
-		/>
+		<footer className="fr-footer" role="contentinfo" id="footer">
+			<div className="fr-container">
+				<div className="fr-footer__body">
+					<div className="fr-footer__brand fr-enlarge-link">
+						<p className="fr-logo">
+							<ConvertContent content={brandTop as any} />
+						</p>
+						<a
+							className="fr-footer__brand-link"
+							href={homeLinkProps.href}
+							title={`Accueil - ${homeLinkProps.title} - République Française`}
+						>
+							<img
+								className="fr-footer__logo"
+								style={{ width: '3.5rem' }}
+								src={operatorLogo?.imgUrl}
+								alt={operatorLogo?.alt}
+							/>
+						</a>
+					</div>
+					<div className="fr-footer__content">
+						<p className="fr-footer__content-desc">{contentDescription}</p>
+						<ul className="fr-footer__content-list">
+							<li className="fr-footer__content-item">
+								<a
+									className="fr-footer__content-link"
+									target="_blank"
+									href="https://legifrance.gouv.fr"
```

Comment:
I get an error eslint on all of the links with `target="_blank"` inciting me to add `rel="noreferrer"`. 
From what I have read, this should have no impact on the user experience, but it may prevent security issues. 
I have pinged the dsfr slack, in the meantime I will add this rel flag to see if it allows us to pass the tests